### PR TITLE
Fix some bare citations

### PR DESCRIPTION
**Pull request by cleong110**
NOTE: 
* I had put _two_ citations in the ASLLVD json on purpose. But it rendered thus:
image

Also fixing this: 

image

#113 is inspired by this

### CODE REVIEW

**Comment by cleong110** [comment]
Hang on, some citations still missing

**Comment by cleong110** [comment]
OK, they WERE added in https://github.com/sign-language-processing/sign-language-processing.github.io/pull/77/files#diff-dbc8abbf7d00f9a6855824bbcdf16d1b6eeb2cef82a718b614e0e30878f0b7d2R3546-R3583, but somehow removed again? 
I'm just going to add them back in.

File: src/datasets/ASLLVD.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "pub": {
     "name": "ASLLVD",
     "year": 2008,
-    "publication": "dataset:athitsos2008american;athitsos2010LargeLexiconIndexingRetrieval",
+    "publication": "dataset:athitsos2008american",
```
Comment (AmitMY) [inline src/datasets/ASLLVD.json:5]: you can have both citations, if you
either:
1. add @ before the second one
2. in the `datasets.js` script, replace ";" with ";@" and ";@@" with ";@"

**Review by AmitMY — APPROVED** [review]

File: src/datasets/ASLLVD.json (hunk 1 above)
Comment (cleong110) [reply src/datasets/ASLLVD.json]: Going with the 1st option for now, we can try the second one another time.

**Pull request merged**